common/services/move.js

```diff
@@ -81,11 +81,11 @@ const moveService = {
   format(data) {
     const booleansAndNulls = ['move_agreed']
     const relationships = [
-      'to_location',
       'from_location',
       'person',
       'prison_transfer_reason',
       'supplier',
+      'to_location',
```

Comment:
Sorting! :heart:

Reply:
I knew Dom was hacking these too - just made sense for us to both sort them so that nothing got overlooked in the merge